rest-api/calamari_rest/serializers/v2.py

```diff
@@ -142,6 +142,8 @@ class Meta:
     public_addr = serializers.CharField(read_only=True, help_text="Public/frontend IP address")
     cluster_addr = serializers.CharField(read_only=True, help_text="Cluster/backend IP address")
     crush_node_ancestry = serializers.Field(help_text="An ordered list of CRUSH node ids that represent a path from the parent node of this OSD up to the root of the tree")
+    backend_partition_path = serializers.CharField(read_only=True, help_text="Full path to the storage partition targeted by this OSD")
```

Comment:
Ok, good deal. I'll get this tested against a version of Ceph that has my required changes as soon as I can, and I'll give you a thumbs up or a thumbs down for a integration then. Sound fair? Until then, if there's anything you want me to change stylistically, or any other tests I should be adding, feel free to let me know.